### ISSUE TITLE
Incorrect XML documentation in LibLog cause my library's build to fail

### ISSUE THREAD

**Issue by Jericho** [damianh/LibLog#219]
I recently upgraded the LibLog in my library to 5.0.5 and I am now getting an exception when building. This exception is caused by incorrect XML document. This is an issue because I treat warnings as errors.

```
obj\Release\netstandard2.0\NuGet\0039BDD62E9C31CEB3F2F82730F75AEDC6743C98\LibLog\5.0.5\LogExtensions.cs(161,26): error CS1572: XML comment has a param tag for 'args', but there is no parameter by that name [C:\_build\Picton.Messaging\Source\Picton.Messaging\Picton.Messaging.csproj]
obj\Release\netstandard2.0\NuGet\0039BDD62E9C31CEB3F2F82730F75AEDC6743C98\LibLog\5.0.5\LogExtensions.cs(163,29): error CS1573: Parameter 'formatParams' has no matching param tag in the XML comment for 'LogExtensions.DebugException(ILog, string, Exception, params object[])' (but other parameters do) [C:\_build\Picton.Messaging\Source\Picton.Messaging\Picton.Messaging.csproj]
```

**Comment by damianh** [comment]
Hi, can you confirm this was recently fixed via #280 please? If so, then I'll ship a patch release soon. Cheers.

**Comment by Jericho** [comment]
I  think you meant to refer to #218. Yes it would resolve the issue I’m experiencing but here are a few observations:

- the PR solves the problem by renaming the parameter in XML documentation, which is perfectly fine, but the standard in all other extension methods seems to be to name this parameter ‘args’ instead of ‘formatargs’. Again, perfectly fine, but for consistency sake I personnalité would have rename the parameter rather than change the xml doc.
- the exact same problem is present in LogEstensions.cs.cpp on line 163
- when an extension method allows logging an exception, the standard signature of this method seems to be ‘this ILog logger, Exception exception, string message, params object[] args’ but the are a few methods where the rider of the exception and the message parameters have been reversed. Not a problem really, I’m just pointing out an inconsistency 
- whenever the exception and the message are reversed, the order of these two parameters in xml doc  does not match

**Comment by Jericho** [comment]
I submitted a PR to address the inconsistencies I previously noted. Specifically: making sure the order of parameters in XML comments matches the order in method signature, making sure the name of the 'format parameter' is consistent across all extension methods, making sure the `.pp` file in in sync with the `.cs` file.

Also, I refrained from standardizing the order of the `exception` and the `message` parameters since that would be a breaking change but I'll be happy to do so if you think it's desirable.
